bindings/rust/src/lib.rs

```diff
@@ -40,6 +40,29 @@ mod sys;
 use std::ffi::CString;
 use std::ptr::NonNull;
 
+#[derive(Eq, PartialEq, Debug, Clone)]
+pub enum Error {
+    MalformedModule(String),
```

Comment:
Would be nicer using `&str` with a lifetime, but maybe better not getting `FIzzyErrorBox` involved into lifetime management?

Reply:
No idea what this mean.